python/ray/tune/suggest/nevergrad.py

```diff
@@ -62,11 +69,18 @@ class NevergradSearch(Searcher):
             "height": tune.uniform(-100, 100),
             "activation": tune.choice(["relu", "tanh"])
         }
+        
+        current_best_params = [{
+            'width': 10,
+            'height': 0,
+            'activation': 0, # The index of "relu"
+        }]
```

Comment:
Let's use our usual quotes here (`"` instead of `'`)

Reply:
done